scripts/uniqueApiKeys.mjs

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+
```

Comment:
read logs from stdin and write output to stdout
so that you can use shell pipes (cat file | uusers.mjs )